packages/rn-tester/RNTesterPods.xcodeproj/project.pbxproj

```diff
@@ -1460,8 +1460,6 @@
 				OTHER_LDFLAGS = (
 					"-ObjC",
 					"-lc++",
-					"-Wl",
-					"-ld_classic",
```

Comment:
I think we can undo these changes and the podfile.lock change. These flags are added by the react native cocoapods install script for some Xcode 15 compatibility (because I think for a while, Xcode 15 used a new linker that broke react native). Not sure if it's still needed so I've just avoided committing it.